### PR TITLE
curl: install zsh completions

### PR DESCRIPTION
**Pull request by qoomon**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/curl.rb [hunk 1]
```diff
@@ -52,6 +52,8 @@ def install
     system "make", "install"
     system "make", "install", "-C", "scripts"
     libexec.install "lib/mk-ca-bundle.pl"
+    
+    zsh_completion.install  "/usr/local/opt/curl/share/zsh/site-functions"  => "_curl"
```
Comment (SMillerDev) [inline Formula/curl.rb:56]: Not every install is `/usr/local`. Please read https://docs.brew.sh/Formula-Cookbook#just-moving-some-files

Comment (qoomon) [reply Formula/curl.rb]: fixed it

**Comment by SMillerDev** [comment]
Please test your changes locally and look up how other formula do this.

**Comment by SMillerDev** [comment]
I'm marking this PR as spam/invalid. It seems not relevant or superflouous. Please review Hactoberfest's quality standard before making additional PRs: https://hacktoberfest.digitalocean.com/details#quality

**Pull request closed without merging**